recipe/meta.yaml

```diff
@@ -9,7 +9,7 @@ source:
   sha256: 14f71c674c9d620c4d5a3483e39917c5ffb7012ecf6f31081d75efa8452a86f6
 
 build:
-  number: 0
+  number: 1
```

Comment:
No need for the build number bump.

Reply:
Ah, I see. Should I change it back to `0` or is the `1` okay, just not necessary?